packages/experiments/src/components/Pagination/Pagination.base.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { BaseComponent, classNamesFunction } from '../../Utilities';
+import { PageNumber } from './PageNumber';
+import { IPaginationProps, IPaginationStyleProps, IPaginationStyles } from './Pagination.types';
+import { FocusZone, FocusZoneDirection } from 'office-ui-fabric-react/lib/FocusZone';
+
+const getClassNames = classNamesFunction<IPaginationStyleProps, IPaginationStyles>();
+
+export class PaginationBase extends BaseComponent<IPaginationProps, {}> {
+  public static defaultProps: Partial<IPaginationProps> = {
+    pageRange: 2,
+    marginPagesDisplayed: 1,
+    omissionLabel: '...'
```

Comment:
Nit and not blocking: You can consider using the actual ellipsis character here. I'm not sure if we have a pattern across the repo.

https://www.toptal.com/designers/htmlarrows/punctuation/horizontal-ellipsis/

Reply:
Let me try a few things and I will try to address it in the future PRs.

---
In reply to: [222068396](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r222068396) [](ancestors = 222068396)